orm/internal/bankexample/state.cosmsos_orm.go

```diff
@@ -0,0 +1,78 @@
+package bankexample
+
+import (
+	context "context"
+
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+
+	ormtable "github.com/cosmos/cosmos-sdk/orm/model/ormtable"
+)
+
+type StateStoreAccessor interface {
+	Open(context.Context) StateStore
+}
+
+func NewStateStoreAccessor() (StateStoreAccessor, error) {
+	panic("TODO")
+}
+
+type StateStore interface {
```

Comment:
I think here it should be:
 ```go
type StateStore interface {
    Balances() BalanceStore
    Supplies() SupplyStore
}
```

Reply:
ex: `stateStore.Balances().Get(ctx, addr, denom)`